tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "incremental": true,
     "types": ["cypress","@testing-library/cypress"]
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "rootDir": "src/",
```

Comment:
unit and integration tests are not in src folder, will this have any impact on ts support for them ?

Reply:
Ah good catch, just added another `tsconfig.json` in the `__tests__` directory to cover this.